cleanlab/multiannotator.py

```diff
@@ -0,0 +1,436 @@
+        TODO: better explanation of this
+    """
+    lqs_consensus_labels = get_label_quality_scores(consensus_labels, pred_probs, **kwargs)
+    frac = pred_probs[range(len(consensus_labels)), (consensus_labels)] / num_annotations
+    quality_of_consensus = frac * lqs_consensus_labels + (1 - frac) * annotator_agreement
+    return quality_of_consensus
+
+
+def compute_consensus_stats(
+    labels_multiannotator: pd.DataFrame,
+    pred_probs: np.array,
+    num_annotations: np.array,
+    consensus_method: str,
+    kwargs: dict = {},
+) -> tuple:
+    """Returns a tuple containing the consensus labels, annotator agreement scores, and quality of consensus
+
+    Parameters
+    ----------
+    labels_multiannotator : pd.DataFrame
+        2D pandas DataFrame of multiple given labels for each example with shape (N, M),
+        where N is the number of examples and M is the number of annotators.
+        For more details, labels in the same format expected by the :py:func:`get_label_quality_scores_multiannotator <cleanlab.multiannotator.get_label_quality_scores_multiannotator>`.
+
+    pred_probs : np.array
+        An array of shape ``(N, K)`` of model-predicted probabilities, ``P(label=k|x)``.
+        For details, predicted probabilities in the same format expected by the :py:func:`get_label_quality_scores_multiannotator <cleanlab.multiannotator.get_label_quality_scores_multiannotator>`.
+
+    num_annotations : np.array
+        An array of shape ``(N,)`` with the number of annotators that have labeled each example.
+
+    method : str (Options: ["majority"])
+        The method used to aggregate labels from multiple annotators into a single consensus label. Default is by using the majority vote.
+        For valid consensus methods, view :py:func:`get_consensus_labels <cleanlab.multiannotator.get_consensus_labels>`
+
+    kwargs : dict
+        Keyword arguments to pass into ``get_label_quality_scores()``.
+
+    Returns
+    ------
+    tuple
+        A tuple of (consensus_labels, annotator_agreement, quality_of_consensus).
+    """
+    # Compute consensus label using method specified
+    consensus_labels = get_consensus_labels(
+        labels_multiannotator=labels_multiannotator,
+        pred_probs=pred_probs,
+        method=consensus_method,
+    )
+
+    # Compute the fraction of annotator agreeing with the consensus labels
+    annotator_agreement = get_annotator_agreement(
+        labels_multiannotator=labels_multiannotator,
+        consensus_labels=consensus_labels,
+    )
+
+    # Compute the label quality scores of the consensus labels
+    quality_of_consensus = get_quality_of_consensus(
+        consensus_labels=consensus_labels,
+        pred_probs=pred_probs,
+        num_annotations=num_annotations,
+        annotator_agreement=annotator_agreement,
+        **kwargs,
+    )
+
+    return (consensus_labels, annotator_agreement, quality_of_consensus)
+
+
+def compute_multiannotator_stats(
```

Comment:
What do you think about changing this to `get_annotator_stats()` ?

Reply:
changed all `compute`s to `get`s !